pkg/templates/output.go

```diff
@@ -0,0 +1,56 @@
+package templates
+
+import (
+	"bufio"
+	"html/template"
+	"os"
+	"os/exec"
+
+	"github.com/fogo-sh/almanac/pkg/content"
+)
+
+func OutputAllPagesToDisk(pages map[string]*content.Page, outputDir string) error {
+	os.RemoveAll(outputDir)
+
+	err := os.MkdirAll(outputDir, 0755)
+	if err != nil {
+		return err
+	}
+
+	allPageTitles := content.AllPageTitles(pages)
+
+	for key, page := range pages {
+		var outputPath = outputDir + key + ".html"
+
+		f, err := os.Create(outputPath)
+		if err != nil {
+			return err
```

Comment:
Can this error, and all other errors in this function, have extra detail attached with `fmt.Errorf`?